galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/src/main/java/dev/galasa/selenium/codesnippet_01.md

```diff
@@ -8,20 +8,22 @@ public ISeleniumManager seleniumManager;
 ```
 
 The code creates an interface to the Selenium Manager which will allow the tester to provision WebPages to test against.
+</details>
 
-### Create a WebPage
```

Comment:
Should this be Open a web page rather than create a web page?

Reply:
changed